builtin_object.go

```diff
@@ -632,6 +646,8 @@ func (r *Runtime) initObject() {
 	o._putProp("keys", r.newNativeFunc(r.object_keys, nil, "keys", nil, 1), true, false, true)
 	o._putProp("setPrototypeOf", r.newNativeFunc(r.object_setPrototypeOf, nil, "setPrototypeOf", nil, 2), true, false, true)
 	o._putProp("values", r.newNativeFunc(r.object_values, nil, "values", nil, 1), true, false, true)
+	o._putProp("fromEntries", r.newNativeFunc(r.object_fromEntries, nil, "fromEntries", nil, 1), true, false, true)
```

Comment:
[q] How come the `fromEntries` were not registered with the props before? It looks like that method existed before for the runtime receiver [here](https://github.com/mongodb-forks/goja/pull/82/files?diff=unified&w=1#diff-9e57b8409307387bf4f821256dee3b79fce1f16d4ab87580a24167ba31de8cceL431).

Reply:
It's strange. Realm branch has [it](https://github.com/mongodb-forks/goja/blob/realm/builtin_object.go#L643) already. And we have two `values` properties in realm branch at Line [634](https://github.com/mongodb-forks/goja/blob/realm/builtin_object.go#L634) and Line [640](https://github.com/mongodb-forks/goja/blob/realm/builtin_object.go#L640). But that should be ok. we just need to clean it.

Reply:
Good catch both `values` and `fromEntries` are registered twice. To answer your question Joey we implemented `fromEntries` before goja had it available so I removed our implementation and used the ones they just added. I did forget to remove registering twice so I will go ahead and fix that!